GPU available: True, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py:1814: PossibleUserWarning: GPU available but not used. Set `accelerator` and `devices` using `Trainer(accelerator='gpu', devices=2)`.
  rank_zero_warn(
Global seed set to 1
Initializing distributed: GLOBAL_RANK: 0, MEMBER: 1/1
----------------------------------------------------------------------------------------------------
distributed_backend=gloo
All distributed processes registered. Starting with 1 processes
----------------------------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/daniel/FYT/comp4332/comp4332_project1/main.py", line 68, in <module>
    main(args)
  File "/home/daniel/FYT/comp4332/comp4332_project1/main.py", line 52, in main
    trainer.fit(model, dm)
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 770, in fit
    self._call_and_handle_interrupt(
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 721, in _call_and_handle_interrupt
    return self.strategy.launcher.launch(trainer_fn, *args, trainer=self, **kwargs)
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/pytorch_lightning/strategies/launchers/subprocess_script.py", line 93, in launch
    return function(*args, **kwargs)
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 811, in _fit_impl
    results = self._run(model, ckpt_path=self.ckpt_path)
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1174, in _run
    self._call_setup_hook()  # allow user to setup lightning_module in accelerator environment
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1492, in _call_setup_hook
    self.datamodule.setup(stage=fn)
  File "/home/daniel/FYT/comp4332/comp4332_project1/data/data_api.py", line 18, in setup
    self.train_set = CommentDataset(self.data_dir, columns=self.columns, split_name='train', embedder= self.embedder)
  File "/home/daniel/FYT/comp4332/comp4332_project1/data/data.py", line 16, in __init__
    self.embedder = Embedder(embedder)
  File "/home/daniel/FYT/comp4332/comp4332_project1/utils/utils.py", line 125, in __init__
    self.embedder=spacy.load(embedder)
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/spacy/__init__.py", line 51, in load
    return util.load_model(
  File "/home/daniel/miniconda3/envs/comp4332proj1/lib/python3.9/site-packages/spacy/util.py", line 427, in load_model
    raise IOError(Errors.E050.format(name=name))
